### PR TITLE
add disclaimer to csv uploader

### PR DESCRIPTION
**Pull request by zdeveloper**
# FRONTEND PULL REQUEST

## Related Issue

- addresses #4340
## Changes Proposed

- Add a disclaimer banner

## Screenshots / Demos

image

### CODE REVIEW

**Comment by lg-king** [comment]
Looks good, just a couple notes on formatting - only need to address if these are quick fixes: 

- Separate first 3 sentences into their own paragraph and italicize (see mockup)
- Make the width a little narrower rather than full width; it looks a little off compared with the text below.

**Comment by zdeveloper** [comment]
> Looks good, just a couple notes on formatting - only need to address if these are quick fixes:
> 
> * Separate first 3 sentences into their own paragraph and italicize (see mockup)
> * Make the width a little narrower rather than full width; it looks a little off compared with the text below.

Done, updated the screenshot

**Review by lg-king — DISMISSED** [review]

File: frontend/src/app/testResults/uploads/Uploads.tsx [hunk 1]
```diff
@@ -140,6 +140,30 @@ const Uploads = () => {
...
+              </em>{" "}
+              <br />
+              <br /> The results uploader allows you to report test results in
+              bulk using a CSV file. While we expect most SimpleReport users
+              will continue to report through the regular process, this feature
+              can serve labs and others who have an information system that can
+              export spreadsheets, such as an EMR. &nbsp;{" "}
+              <LinkWithQuery to="/results/upload/submit/guide">
+                <strong>Learn more about how it works</strong>
+              </LinkWithQuery>
```
Comment (lauraykerr) [inline frontend/src/app/testResults/uploads/Uploads.tsx:163]:
```suggestion
              export spreadsheets, such as an EMR. {" "}
              <LinkWithQuery to="/results/upload/submit/guide">
                <strong>Learn more about how it works</strong>
              </LinkWithQuery>.
```
Looks great! Made one small suggestion to remove extra space before the link and add a period at the end.

File: frontend/src/app/testResults/uploads/Uploads.tsx [hunk 2]
```diff
@@ -140,6 +140,30 @@ const Uploads = () => {
...
+                to update and improve it for our users. Though the feature is in
+                beta, it will route all the results you submit to the
+                appropriate public health department(s).
+              </em>{" "}
+              <br />
+              <br /> The results uploader allows you to report test results in
+              bulk using a CSV file. While we expect most SimpleReport users
+              will continue to report through the regular process, this feature
+              can serve labs and others who have an information system that can
+              export spreadsheets, such as an EMR. &nbsp;{" "}
```
Comment (lauraykerr) [inline frontend/src/app/testResults/uploads/Uploads.tsx:160]:
```suggestion
              can serve labs and others with an information system that
              exports spreadsheets, such as an EMR. &nbsp;{" "}
```
Also editing my own suggested text now that I'm seeing it in context.

Comment (zdeveloper) [reply frontend/src/app/testResults/uploads/Uploads.tsx]: changes merged

Comment (zdeveloper) [reply frontend/src/app/testResults/uploads/Uploads.tsx]: updated the screenshot

**Review by mehansen — APPROVED** [review]
lgtm!

what's the process for the UI Review steps in the PR checks? I see it's unresolved but that the check is not required

**Review by lauraykerr — APPROVED** [review]

**Pull request merged**